.github/actions/update-rpm-config/action.yml

```diff
@@ -0,0 +1,109 @@
+name: "update-rpm-config"
+description: "Set current version of agent in rpm config using API."
+inputs:
+  agent-language:
+    description: "Language agent to configure (eg. python)"
+    required: true
+    default: "python"
+  target-system:
+    description: "Target System: prod|staging|all"
+    required: true
+    default: "all"
+  agent-version:
+    description: "3-4 digit agent version number (eg. 1.2.3) with optional leading v (ignored)"
+    required: true
+  dry-run:
+    description: "Dry Run"
+    required: true
+    default: "false"
+  production-api-key:
+    description: "API key for New Relic Production"
+    required: false
+  staging-api-key:
+    description: "API key for New Relic Staging"
+    required: false
+
+runs:
+  using: "composite"
+  steps:
+    - name: Trim potential leading v from agent version
```

Comment:
Why not just pass the version without the preceding v?

Reply:
It's easier to put the normalization logic inside the "function" (action) than to rely on the format of the input being exactly correct. Especially if you just pass in `${{github.ref_name}}` which comes with a `v` prefix for us, it prevents you from having to run a step of logic in the workflow to trim that before using it as input.